guardrails/run/async_runner.py

```diff
@@ -144,7 +144,7 @@ async def async_run(
 
         return call_log
 
-    @async_trace(name="step")
+    @async_trace(name="async_step")
```

Comment:
We need to maybe also revert this at some point? But currently it breaks if it uses 'step' due to some async issues